lib/screen/editorial/article/article_detail.dart

```diff
@@ -67,23 +69,35 @@ class _ArticleDetailPageState extends State<ArticleDetailPage> {
   }
 
   _scrollListener() {
+    _timer?.cancel();
+    _scrollTimer?.cancel();
+
     if (_controller.offset > 5) {
       setState(() {
         _showHeader = false;
       });
-      _timer?.cancel();
-      _timer = Timer(const Duration(seconds: 5), () {
-        if (mounted) {
+
+      final difference = _controller.offset - _lastOffset;
+      _scrollTimer = Timer(const Duration(milliseconds: 300), () {
+        if (difference > -5 && difference < 0) {
```

Comment:
Do you mean `difference < -5`?

Reply:
No, it for slightly scroll up case, show header